engine/core/Engine.cpp

```diff
@@ -230,3 +236,28 @@ bool Engine::isStarted() {
 double Engine::getDeltaTime() {
     return Engine::currentTime - Engine::lastTime;
 }
+
+std::string Engine::getConfigDir() {
```

Comment:
Can the result of this function be cached so future runs just return the value already calculated? A static variable would work great

Reply:
difficulty comes from the fact we're always depending on the user being the same. we don't want to pull from the wrong user directory do we?